ogame/__init__.py

```diff
@@ -1310,6 +1295,30 @@ def build(self, what, id):
                     'menge': amount}
         )
 
+    def deconstruct(self, what, id):
```

Comment:
really clean implementation. easy to understand and how it is structured good job